GUI/community-version.py

```diff
@@ -120,6 +123,7 @@ def handle_image_print(image_ascii, color, store):
 def write_to_txtfile(txt):
     try:
         with open("output.txt", "w") as text_file:
+            print(pyfiglet.figlet_format("Welcome to ASCII ART Generator"))
```

Comment:
I think that if you're going to print a welcome message, it makes more sense to put it at the beginning of the program, not inside a function that is responsible for writing to a text file.